app/models/measure_type.rb

```diff
@@ -28,10 +28,20 @@ def description
     translated_description || attributes['description']
   end
 
+  def description_only
+    attributes['description']
+  end
+
   def supplementary_unit_import_only?
     id.in?(SUPPLEMENTARY_IMPORT_ONLY_MEASURE_TYPES)
   end
 
+  def md_details
+    File.read("db/measure_type_detail_texts/#{TradeTariffFrontend::ServiceChooser.service_name}/#{id}.md")
+  rescue Errno::ENOENT
```

Comment:
I don't like exceptions to control decisions generally. How do you feel about just explicitly checking if the file exists?